ci-operator/step-registry/openshift-qe/node-density-cni/openshift-qe-node-density-cni-commands.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+oc config view
+oc projects
+python --version
+pushd /tmp
+python -m virtualenv ./venv_qe
+source ./venv_qe/bin/activate
+
+ES_PASSWORD=$(cat "/secret/password")
+ES_USERNAME=$(cat "/secret/username")
+
+git clone https://github.com/cloud-bulldozer/e2e-benchmarking
+pushd e2e-benchmarking/workloads/kube-burner
```

Comment:
Let's go ahead and use the wrapper version of this workload like: https://github.com/openshift/release/blob/e1e41f889bf7d99e4d4cbc604d35841912f2293a/ci-operator/step-registry/openshift-qe/node-density/openshift-qe-node-density-commands.sh#L18

Reply:
be sure to update the parameters to look similar to the variables in the referenced command as well